knowledge-base/pass-parameters-to-html5-report-viewer-from-a-views-model.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Pass parameters to the HTML5 Report Viewer from the Model of a View
+description: "Learn the specifics of passing a parameter to the HTML5 Report Viewer from the Model of a View"

```

Comment:
Not sure if the description can have quotation marks, I would recommend avoiding it